networkx/convert_matrix.py

```diff
@@ -397,14 +400,16 @@ def from_pandas_edgelist(
     attr_col_headings = []
     attribute_data = []
     if edge_attr is True:
-        attr_col_headings = [c for c in df.columns if c not in reserved_columns]
+        attr_col_headings = [
+            c for c in df.columns if c not in reserved_columns
+        ]
     elif isinstance(edge_attr, (list, tuple)):
         attr_col_headings = edge_attr
     else:
         attr_col_headings = [edge_attr]
     if len(attr_col_headings) == 0:
         raise nx.NetworkXError(
-            f"Invalid edge_attr argument: No columns found with name: {attr_col_headings}"
+            f"Invalid edge_attr argument: No columns found with name: {attr_col_headings}"  # noqa: E501
```

Comment:
Can we take out ```noqa``` stuff, since we're moving to relying on black?
I don't know the answer, but it'd be nice not to have distractions in the code.

Reply:
If you replace
```
f"Invalid edge_attr argument: No columns found with name: {attr_col_headings}"
```
with
```
f"Invalid edge_attr: No columns found with name: {attr_col_headings}"
```
it won't get wrapped.

Reply:
That's true. For consistency, I've modified all of the other lines in the same function that carry the term `argument` in the error message.